src/core/qgsapplication.h

```diff
@@ -449,6 +449,14 @@ class CORE_EXPORT QgsApplication : public QApplication
      */
     static QString locale();
 
+    /**
+     * Sets the QGIS locale - used mainly by 3rd party apps and tests.
+     * In QGIS this is internally triggered by the application in startup.
+     *
+     * \since QGIS 3.22
```

Comment:
3.24

Reply:
I tought that after backport this is available in 3.22. I will cherry-pick these commits (not the first one) to backport branch. What do you think, which one should we use?

Reply:
I forgot that the backport was already discussed in the original PR.
Let's go for 3.22.2 or 3.22.3

Reply:
Alright, I will change those to 3.22.2.